steely/plugins/linden.py

```diff
@@ -194,7 +194,7 @@ def give_cmd(bot, message_parts, author_id, thread_id, thread_type):
     if not reciever:
         reciever = create_user(reciever_model.uid, reciever_model.first_name)
     if not sender:
-        sender = create_user(author_id, sender_model.first_name)
+        bot.sendMessage('Sorry! Lindens are invite only!!!!', thread_id=thread_id, thread_type=thread_type)
```

Comment:
Shouldn't it return here or sth

Reply:
definitely should but im an idiot so i forgot, ty